[{"line":1,"elements":[{"start_timestamp":"2025-07-15T12:48:51.810Z","line":4,"name":"","description":"","id":"forgot-password;","type":"scenario","keyword":"Scenario","steps":[],"tags":[{"name":"@FP"}]},{"start_timestamp":"2025-07-15T12:48:51.891Z","line":11,"name":"","description":"","id":"forgot-password;","type":"scenario","keyword":"Scenario","steps":[],"tags":[{"name":"@FP1"}]}],"name":"Forgot Password","description":"","id":"forgot-password","keyword":"Feature","uri":"file:src/test/resources/features/ForgotPassword.feature","tags":[]},{"line":1,"elements":[{"start_timestamp":"2025-07-15T12:48:51.897Z","before":[{"result":{"duration":24398747300,"status":"passed"},"match":{"location":"stepDefinitions.Hooks.setUp()"}}],"line":4,"name":"Login to the application with valid data and logout of the application","description":"","id":"login-and-logout-feature;login-to-the-application-with-valid-data-and-logout-of-the-application","after":[{"result":{"duration":1123524700,"status":"passed"},"match":{"location":"stepDefinitions.Hooks.tearDown()"}}],"type":"scenario","keyword":"Scenario","steps":[{"result":{"duration":3325648000,"status":"passed"},"line":5,"name":"Launch the app","match":{"location":"stepDefinitions.TC_1_LoginWithValidCredentails.Launch_the_app()"},"keyword":"Given "},{"result":{"duration":4260165300,"status":"passed"},"line":6,"name":"User enters valid username and password","match":{"location":"stepDefinitions.TC_1_LoginWithValidCredentails.enter_credentials()"},"keyword":"When "},{"result":{"duration":2240615500,"status":"passed"},"line":7,"name":"User tap on the login button","match":{"location":"stepDefinitions.TC_1_LoginWithValidCredentails.User_tap_on_the_login_button()"},"keyword":"Then "},{"result":{"duration":11408800700,"status":"passed"},"line":8,"name":"User tap on profile icon","match":{"location":"stepDefinitions.TC_1_LoginWithValidCredentails.User_tap_on_profile_icon()"},"keyword":"And "},{"result":{"duration":3112016600,"status":"passed"},"line":9,"name":"User tap on logout button","match":{"location":"stepDefinitions.TC_1_LoginWithValidCredentails.click_tap_on_logout_button()"},"keyword":"And "}],"tags":[{"name":"@P1"}]},{"start_timestamp":"2025-07-15T12:49:41.791Z","before":[{"result":{"duration":6925907200,"status":"passed"},"match":{"location":"stepDefinitions.Hooks.setUp()"}}],"line":12,"name":"Try to login without data","description":"","id":"login-and-logout-feature;try-to-login-without-data","after":[{"result":{"duration":856270500,"status":"passed"},"match":{"location":"stepDefinitions.Hooks.tearDown()"}}],"type":"scenario","keyword":"Scenario","steps":[{"result":{"duration":3179645600,"status":"passed"},"line":13,"name":"Launch the app","match":{"location":"stepDefinitions.TC_1_LoginWithValidCredentails.Launch_the_app()"},"keyword":"Given "},{"result":{"duration":1392816800,"status":"passed"},"line":14,"name":"User tap on the login button","match":{"location":"stepDefinitions.TC_1_LoginWithValidCredentails.User_tap_on_the_login_button()"},"keyword":"And "},{"result":{"duration":60458600,"status":"passed"},"line":15,"name":"Validate error message displayed below email text field","match":{"location":"stepDefinitions.Tc_2.validate_error_message_displayed_below_email_text_field()"},"keyword":"Then "},{"result":{"duration":44716900,"status":"passed"},"line":16,"name":"Validate error message displayed below password text field","match":{"location":"stepDefinitions.Tc_2.validate_error_message_displayed_below_password_text_field()"},"keyword":"And "}],"tags":[{"name":"@P2"}]}],"name":"Login and Logout Feature","description":"","id":"login-and-logout-feature","keyword":"Feature","uri":"file:src/test/resources/features/Login_Logout.feature","tags":[]},{"line":1,"elements":[{"start_timestamp":"2025-07-15T12:49:54.263Z","line":2,"name":"Failure while executing Cucumber","description":"","id":"failure;failure-while-executing-cucumber","type":"scenario","keyword":"Scenario","steps":[{"result":{"duration":0,"status":"passed"},"line":3,"name":"Cucumber failed while executing","match":{"arguments":[],"location":"io.cucumber.core.Failure.failure_while_executing_cucumber()"},"keyword":"When "},{"result":{"duration":0,"error_message":"io.cucumber.core.exception.CompositeCucumberException: There were 2 exceptions. The details are in the stacktrace below.\r\n\tat io.cucumber.core.runtime.RethrowingThrowableCollector.getThrowable(RethrowingThrowableCollector.java:57)\r\n\tat io.cucumber.core.runtime.CucumberExecutionContext.getThrowable(CucumberExecutionContext.java:108)\r\n\tat io.cucumber.core.runtime.CucumberExecutionContext.finishTestRun(CucumberExecutionContext.java:103)\r\n\tat io.cucumber.testng.TestNGCucumberRunner.finish(TestNGCucumberRunner.java:148)\r\n\tat io.cucumber.testng.AbstractTestNGCucumberTests.tearDownClass(AbstractTestNGCucumberTests.java:57)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeAfterClassConfigurations(TestMethodWorker.java:238)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:223)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:137)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)\r\n\tSuppressed: java.lang.IllegalArgumentException: Test name must not be null or empty\r\n\t\tat com.aventstack.extentreports.util.Assert.notEmpty(Assert.java:12)\r\n\t\tat com.aventstack.extentreports.ExtentTest.<init>(ExtentTest.java:96)\r\n\t\tat com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:153)\r\n\t\tat com.aventstack.extentreports.cucumber.adapter.ExtentCucumberAdapter.createTestCase(ExtentCucumberAdapter.java:424)\r\n\t\tat com.aventstack.extentreports.cucumber.adapter.ExtentCucumberAdapter.handleTestCaseStarted(ExtentCucumberAdapter.java:163)\r\n\t\tat com.aventstack.extentreports.cucumber.adapter.ExtentCucumberAdapter$3.receive(ExtentCucumberAdapter.java:107)\r\n\t\tat com.aventstack.extentreports.cucumber.adapter.ExtentCucumberAdapter$3.receive(ExtentCucumberAdapter.java:104)\r\n\t\tat io.cucumber.core.eventbus.AbstractEventPublisher.send(AbstractEventPublisher.java:51)\r\n\t\tat io.cucumber.core.eventbus.AbstractEventBus.send(AbstractEventBus.java:12)\r\n\t\tat io.cucumber.core.runtime.SynchronizedEventBus.send(SynchronizedEventBus.java:47)\r\n\t\tat io.cucumber.core.runtime.ThreadLocalRunnerSupplier$LocalEventBus.send(ThreadLocalRunnerSupplier.java:62)\r\n\t\tat io.cucumber.core.runner.TestCase.emitTestCaseStarted(TestCase.java:213)\r\n\t\tat io.cucumber.core.runner.TestCase.run(TestCase.java:74)\r\n\t\tat io.cucumber.core.runner.Runner.runPickle(Runner.java:77)\r\n\t\tat io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:135)\r\n\t\tat io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:136)\r\n\t\tat io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)\r\n\t\tat io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:136)\r\n\t\tat io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:132)\r\n\t\tat io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:35)\r\n\t\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\t\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\t\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\t\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\t\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\t\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\t\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\t\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\t\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\t\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\t\t... 16 more\r\n\tSuppressed: java.lang.IllegalArgumentException: Test name must not be null or empty\r\n\t\tat com.aventstack.extentreports.util.Assert.notEmpty(Assert.java:12)\r\n\t\tat com.aventstack.extentreports.ExtentTest.<init>(ExtentTest.java:96)\r\n\t\tat com.aventstack.extentreports.ExtentTest.createNode(ExtentTest.java:153)\r\n\t\tat com.aventstack.extentreports.cucumber.adapter.ExtentCucumberAdapter.createTestCase(ExtentCucumberAdapter.java:424)\r\n\t\tat com.aventstack.extentreports.cucumber.adapter.ExtentCucumberAdapter.handleTestCaseStarted(ExtentCucumberAdapter.java:163)\r\n\t\tat com.aventstack.extentreports.cucumber.adapter.ExtentCucumberAdapter$3.receive(ExtentCucumberAdapter.java:107)\r\n\t\tat com.aventstack.extentreports.cucumber.adapter.ExtentCucumberAdapter$3.receive(ExtentCucumberAdapter.java:104)\r\n\t\tat io.cucumber.core.eventbus.AbstractEventPublisher.send(AbstractEventPublisher.java:51)\r\n\t\tat io.cucumber.core.eventbus.AbstractEventBus.send(AbstractEventBus.java:12)\r\n\t\tat io.cucumber.core.runtime.SynchronizedEventBus.send(SynchronizedEventBus.java:47)\r\n\t\tat io.cucumber.core.runtime.ThreadLocalRunnerSupplier$LocalEventBus.send(ThreadLocalRunnerSupplier.java:62)\r\n\t\tat io.cucumber.core.runner.TestCase.emitTestCaseStarted(TestCase.java:213)\r\n\t\tat io.cucumber.core.runner.TestCase.run(TestCase.java:74)\r\n\t\tat io.cucumber.core.runner.Runner.runPickle(Runner.java:77)\r\n\t\tat io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:135)\r\n\t\tat io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:136)\r\n\t\tat io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)\r\n\t\tat io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:136)\r\n\t\tat io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:132)\r\n\t\tat io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:35)\r\n\t\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\t\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\t\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\t\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\t\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\t\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\t\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\t\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\t\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\t\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\t\t... 16 more\r\n","status":"failed"},"line":4,"name":"Cucumber will report this error:","match":{"arguments":[],"location":"io.cucumber.core.Failure.cucumber_reports_this_error()"},"keyword":"Then "}]}],"name":"Test run failed","description":"There were errors during the execution","id":"failure","keyword":"Feature","uri":"classpath:io/cucumber/core/failure.feature","tags":[]}]